docker/all-in-one/Dockerfile

```diff
@@ -0,0 +1,233 @@
+ADD "https://supabase-public-artifacts-bucket.s3.amazonaws.com/supabase-admin-api/v${adminapi_release}/supabase-admin-api_${adminapi_release}_linux_${TARGETARCH}.tar.gz" /tmp/supabase-admin-api.tar.gz
+RUN tar -xvf /tmp/supabase-admin-api.tar.gz -C /tmp && \
+    rm -rf /tmp/supabase-admin-api.tar.gz
+
+####################
+# Install admin mgr
+####################
+FROM base as adminmgr
+ARG adminmgr_release
+ADD "https://supabase-public-artifacts-bucket.s3.amazonaws.com/admin-mgr/v${adminmgr_release}/admin-mgr_${adminmgr_release}_linux_${TARGETARCH}.tar.gz" /tmp/admin-mgr.tar.gz
+RUN tar -xvf /tmp/admin-mgr.tar.gz -C /tmp && \
+    rm -rf /tmp/admin-mgr.tar.gz
+
+####################
+# Install Prometheus Exporter
+####################
+FROM base as exporter
+ARG postgres_exporter_release
+ADD "https://github.com/prometheus-community/postgres_exporter/releases/download/v${postgres_exporter_release}/postgres_exporter-${postgres_exporter_release}.linux-${TARGETARCH}.tar.gz" /tmp/postgres_exporter.tar.gz
+RUN tar -xvf /tmp/postgres_exporter.tar.gz -C /tmp --strip-components 1 && \
+    rm -rf /tmp/postgres_exporter.tar.gz
+
+####################
+# Install vector
+####################
+FROM base as vector
+ARG vector_release
+ADD "https://packages.timber.io/vector/${vector_release}/vector_${vector_release}-1_${TARGETARCH}.deb" /tmp/vector.deb
+
+####################
+# Install supervisord
+####################
+FROM base as supervisor
+# Download pre-built packages
+RUN apt-get update -y && apt-get install -y --no-install-recommends --download-only \
+    supervisor \
+    && rm -rf /var/lib/apt/lists/*
+RUN mv /var/cache/apt/archives/*.deb /tmp/
+
+####################
+# Create the final image for production
+####################
+FROM base as production
+
+# Copy dependencies from previous build stages
+COPY --from=pgbouncer /tmp/*.deb /tmp/
+COPY --from=vector /tmp/*.deb /tmp/
+COPY --from=kong /tmp/*.deb /tmp/
+COPY --from=supervisor /tmp/*.deb /tmp/
+
+# Install runtime dependencies
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    /tmp/*.deb \
+    # For health check
+    curl \
+    # For parsing init payload
+    jq \
+    # Security tools
+    fail2ban \
+    # sudo
+    sudo \
+    # reading files using cat sucks
+    less \
+    && rm -rf /var/lib/apt/lists/* /tmp/*
+
+# Copy single binary dependencies
+COPY --from=pgrst /bin/postgrest /opt/
+COPY --from=gotrue /usr/local/bin/gotrue /opt/gotrue/
+COPY --from=gotrue /usr/local/etc/gotrue /opt/gotrue/
+COPY --from=adminapi /tmp/supabase-admin-api /opt/
+COPY --chown=root:root --from=adminmgr /tmp/admin-mgr /usr/bin/
+COPY --from=exporter /tmp/postgres_exporter /opt/postgres_exporter/
+COPY docker/all-in-one/opt/postgres_exporter /opt/postgres_exporter/
+
+# Scripts for adminapi
+COPY ansible/files/admin_api_scripts /root 
+COPY --chown=adminapi:adminapi docker/all-in-one/etc/adminapi /etc/adminapi
+COPY docker/all-in-one/etc/sudoers.d /etc/sudoers.d/
```

Comment:
Not sure about hardcoding the path here - was thinking of pulling the path into a build arg but then you still won't be able to build the image without the files in this repo

Reply:
that seems ok? why would you need to be able to build w/o the files from here?